jdk-postgres/8-15/Dockerfile

```diff
@@ -85,7 +85,7 @@ RUN set -ex; \
 ENV PG_MAJOR 15
 ENV PATH $PATH:/usr/lib/postgresql/$PG_MAJOR/bin
 
-ENV PG_VERSION 15.6-1.pgdg110+2
+ENV PG_VERSION 15.7-1.pgdg110+1
```

Comment:
@choplin 
PostgreSQL updated the Debian package to 15.7 for version 15. I think it should be safe because it's a minor version but please check it just in case 🙇